### ISSUE TITLE
[Bug] zos_job_submit not filtering ddname output

### ISSUE THREAD

**Issue by itsBryantP** [ansible-collections/ibm_zos_core#308]
##### SUMMARY

When using version `1.4.0-beta.1` of the core collection and the `zos_job_submit` module, the returned output repeats the entire joblog for each ddname entry rather than the individual output streams. Running the same module with version `1.3.1` produces the correct behavior.  

##### COMPONENT NAME
`zos_job_submit`

##### ANSIBLE VERSION
```
ansible [core 2.11.6] 
  config file = /Users/bpanyar/github/z_ansible_collections_samples_demo/z_ansible_collections_samples/zos_concepts/data_sets/copy_edit_submit/ansible.cfg
  configured module search path = ['/Users/bpanyar/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/Cellar/ansible/4.8.0/libexec/lib/python3.10/site-packages/ansible
  ansible collection location = /Users/bpanyar/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/local/bin/ansible
  python version = 3.10.0 (default, Oct 13 2021, 06:45:00) [Clang 13.0.0 (clang-1300.0.29.3)]
  jinja version = 3.0.2
  libyaml = True
```

##### SPECIFY THE Z OPEN AUTOMATION UTILITIES VERSION
ZOAU Version 1.1.0

##### ENVIRONMENT
z/OS 2.3

##### STEPS TO REPRODUCE
```
- name: "Submit the JCL {{ tmp_data_set }}(UPTIME)"
      zos_job_submit:
        src: "{{ tmp_data_set }}(UPTIME)"
        location: DATA_SET
        wait: true
      register: result_submit_uptime

    - name: "Response for submitting JCL {{ tmp_data_set }}(UPTIME)"
      debug:
        var: result_submit_uptime
```

##### EXPECTED RESULTS
```
TASK [Response for submitting JCL BPANYAR.P0003771.T0557691.C0000000(UPTIME)] **********************************************************************************
ok: [zos_host] => {
    "result_submit_uptime": {
        "changed": true,
        "duration": 1,
        "failed": false,
        "jobs": [
            {
                "class": "A",
                "content_type": "JOB",
                "ddnames": [
                    {
                        "byte_count": "1181",
                        "content": [
                            "1                    J E S 2  J O B  L O G  --  S Y S T E M  I P O 1  --  N O D E  T I V L P 0 2         ",
                            "0 ",
                            " 20.22.21 JOB01146 ---- WEDNESDAY, 08 DEC 2021 ----",
                            " 20.22.21 JOB01146  IRR010I  USERID BPANYAR  IS ASSIGNED TO THIS JOB.",
                            " 20.22.21 JOB01146  $HASP375 JES2     ESTIMATED  LINES EXCEEDED",
                            " 20.22.21 JOB01146  ICH70001I BPANYAR  LAST ACCESS AT 20:22:20 ON WEDNESDAY, DECEMBER 8, 2021",
                            " 20.22.21 JOB01146  $HASP375 UPTIME   ESTIMATED  LINES EXCEEDED",
                            " 20.22.21 JOB01146  $HASP373 UPTIME   STARTED - INIT K    - CLASS A        - SYS IPO1",
                            " 20.22.21 JOB01146  -                                                --TIMINGS (MINS.)--            ----PAGING COUNTS---",
                            " 20.22.21 JOB01146  -JOBNAME  STEPNAME PROCSTEP    RC   EXCP   CONN    TCB    SRB  CLOCK   SERV  PG  PAGE  SWAP   VIO SWAPS",
                            " 20.22.21 JOB01146  -UPTIME   UPTIME               00     40      1    .00    .00     .0    130   0     0     0     0     0",
                            " 20.22.21 JOB01146  -UPTIME   ENDED.  NAME-UPTIME - JRM         TOTAL TCB CPU TIME=   .00  TOTAL ELAPSED TIME=    .0",
                            " 20.22.21 JOB01146  $HASP395 UPTIME   ENDED - RC=0000",
                            "0------ JES2 JOB STATISTICS ------",
                            "-  08 DEC 2021 JOB EXECUTION DATE",
                            "-           12 CARDS READ",
                            "-           58 SYSOUT PRINT RECORDS",
                            "-            0 SYSOUT PUNCH RECORDS",
                            "-            7 SYSOUT SPOOL KBYTES",
                            "-         0.00 MINUTES EXECUTION TIME"
                        ],
                        "ddname": "JESMSGLG",
                        "id": "2",
                        "procstep": "",
                        "record_count": "20",
                        "stepname": "JES2"
                    },
                    {
                        "byte_count": "546",
                        "content": [
                            "         1 //UPTIME    JOB (T043JM,JM00,1,0,0,0),'UPTIME - JRM',                   JOB01146",
                            "           //             MSGCLASS=X,MSGLEVEL=1,NOTIFY=&SYSUID                             ",
                            "           //*                                                                             ",
                            "           //* PRINT USS COMMAND ON JOB OUTPUT                                             ",
                            "           //*                                                                             ",
                            "           IEFC653I SUBSTITUTION JCL - (T043JM,JM00,1,0,0,0),'UPTIME - JRM',MSGCLASS=X,MSGLEVEL=1,NOTIFY=BPANYAR",
                            "         2 //UPTIME  EXEC PGM=BPXBATCH                                                     ",
                            "         3 //STDPARM DD *                                                                  ",
                            "         4 //STDIN  DD DUMMY                                                               ",
                            "         5 //STDOUT DD SYSOUT=*                                                            ",
                            "         6 //STDERR DD SYSOUT=*                                                            ",
                            "         7 //                                                                              "
                        ],
                        "ddname": "JESJCL",
                        "id": "3",
                        "procstep": "",
                        "record_count": "12",
                        "stepname": "JES2"
                    },
                    {
                        "byte_count": "1194",
                        "content": [
                            " ICH70001I BPANYAR  LAST ACCESS AT 20:22:20 ON WEDNESDAY, DECEMBER 8, 2021",
                            " IEFA111I UPTIME IS USING THE FOLLOWING JOB RELATED SETTINGS:",
                            "          SWA=BELOW,TIOT SIZE=32K,DSENQSHR=DISALLOW,GDGBIAS=JOB",
                            " IEF236I ALLOC. FOR UPTIME UPTIME",
                            " IEF237I JES2 ALLOCATED TO STDPARM",
                            " IEF237I DMY  ALLOCATED TO STDIN",
                            " IEF237I JES2 ALLOCATED TO STDOUT",
                            " IEF237I JES2 ALLOCATED TO STDERR",
                            " IEF142I UPTIME UPTIME - STEP WAS EXECUTED - COND CODE 0000",
                            " IEF285I   BPANYAR.UPTIME.JOB01146.D0000101.?           SYSIN         ",
                            " IEF285I   BPANYAR.UPTIME.JOB01146.D0000102.?           SYSOUT        ",
                            " IEF285I   BPANYAR.UPTIME.JOB01146.D0000103.?           SYSOUT        ",
                            " IEF373I STEP/UPTIME  /START 2021342.2022",
                            " IEF032I STEP/UPTIME  /STOP  2021342.2022 ",
                            "         CPU:     0 HR  00 MIN  00.00 SEC    SRB:     0 HR  00 MIN  00.00 SEC    ",
                            "         VIRT:   168K  SYS:   248K  EXT:      132K  SYS:    11744K",
                            "         ATB- REAL:                  1044K  SLOTS:                     0K",
                            "              VIRT- ALLOC:      11M SHRD:       0M",
                            " IEF375I  JOB/UPTIME  /START 2021342.2022",
                            " IEF033I  JOB/UPTIME  /STOP  2021342.2022 ",
                            "         CPU:     0 HR  00 MIN  00.00 SEC    SRB:     0 HR  00 MIN  00.00 SEC    "
                        ],
                        "ddname": "JESYSMSG",
                        "id": "4",
                        "procstep": "",
                        "record_count": "21",
                        "stepname": "JES2"
                    },
                    {
                        "byte_count": "108",
                        "content": [
                            " 08:22PM  up 31 day(s), 05:49,  1 users,  load average: 0.00, 0.00, 0.00"
                        ],
                        "ddname": "STDOUT",
                        "id": "102",
                        "procstep": "",
                        "record_count": "1",
                        "stepname": "UPTIME"
                    }
                ],
                "job_id": "JOB01146",
                "job_name": "UPTIME",
                "owner": "BPANYAR",
                "ret_code": {
                    "code": 0,
                    "msg": "CC 0000",
                    "msg_code": "0000",
                    "msg_txt": ""
                },
                "subsystem": "IPO1",
                "system": ""
            }
        ],
        "message": {
            "stdout": "Submit JCL operation succeeded."
        }
    }
}
```

##### ACTUAL RESULTS
```
TASK [Response for submitting JCL BPANYAR.P0003812.T0278773.C0000000(UPTIME)] **********************************************************************************
ok: [zos_host] => {
    "result_submit_uptime": {
        "changed": true,
        "duration": 2,
        "failed": false,
        "job_id": "JOB01151",
        "jobs": [
            {
                "class": "A",
                "content_type": "JOB",
                "ddnames": [
                    {
                        "byte_count": "1181",
                        "content": [
                            "1                    J E S 2  J O B  L O G  --  S Y S T E M  I P O 1  --  N O D E  T I V L P 0 2         ",
                            "0 ",
                            " 20.24.59 JOB01151 ---- WEDNESDAY, 08 DEC 2021 ----",
                            " 20.24.59 JOB01151  IRR010I  USERID BPANYAR  IS ASSIGNED TO THIS JOB.",
                            " 20.24.59 JOB01151  $HASP375 JES2     ESTIMATED  LINES EXCEEDED",
                            " 20.24.59 JOB01151  ICH70001I BPANYAR  LAST ACCESS AT 20:24:59 ON WEDNESDAY, DECEMBER 8, 2021",
                            " 20.24.59 JOB01151  $HASP375 UPTIME   ESTIMATED  LINES EXCEEDED",
                            " 20.24.59 JOB01151  $HASP373 UPTIME   STARTED - INIT K    - CLASS A        - SYS IPO1",
                            " 20.24.59 JOB01151  -                                                --TIMINGS (MINS.)--            ----PAGING COUNTS---",
                            " 20.24.59 JOB01151  -JOBNAME  STEPNAME PROCSTEP    RC   EXCP   CONN    TCB    SRB  CLOCK   SERV  PG  PAGE  SWAP   VIO SWAPS",
                            " 20.24.59 JOB01151  -UPTIME   UPTIME               00     40      1    .00    .00     .0    122   0     0     0     0     0",
                            " 20.24.59 JOB01151  -UPTIME   ENDED.  NAME-UPTIME - JRM         TOTAL TCB CPU TIME=   .00  TOTAL ELAPSED TIME=    .0",
                            " 20.24.59 JOB01151  $HASP395 UPTIME   ENDED - RC=0000",
                            "0------ JES2 JOB STATISTICS ------",
                            "-  08 DEC 2021 JOB EXECUTION DATE",
                            "-           12 CARDS READ",
                            "-           58 SYSOUT PRINT RECORDS",
                            "-            0 SYSOUT PUNCH RECORDS",
                            "-            7 SYSOUT SPOOL KBYTES",
                            "-         0.00 MINUTES EXECUTION TIME",
                            "         1 //UPTIME    JOB (T043JM,JM00,1,0,0,0),'UPTIME - JRM',                   JOB01151",
                            "           //             MSGCLASS=X,MSGLEVEL=1,NOTIFY=&SYSUID                             ",
                            "           //*                                                                             ",
                            "           //* PRINT USS COMMAND ON JOB OUTPUT                                             ",
                            "           //*                                                                             ",
                            "           IEFC653I SUBSTITUTION JCL - (T043JM,JM00,1,0,0,0),'UPTIME - JRM',MSGCLASS=X,MSGLEVEL=1,NOTIFY=BPANYAR",
                            "         2 //UPTIME  EXEC PGM=BPXBATCH                                                     ",
                            "         3 //STDPARM DD *                                                                  ",
                            "         4 //STDIN  DD DUMMY                                                               ",
                            "         5 //STDOUT DD SYSOUT=*                                                            ",
                            "         6 //STDERR DD SYSOUT=*                                                            ",
                            "         7 //                                                                              ",
                            " ICH70001I BPANYAR  LAST ACCESS AT 20:24:59 ON WEDNESDAY, DECEMBER 8, 2021",
                            " IEFA111I UPTIME IS USING THE FOLLOWING JOB RELATED SETTINGS:",
                            "          SWA=BELOW,TIOT SIZE=32K,DSENQSHR=DISALLOW,GDGBIAS=JOB",
                            " IEF236I ALLOC. FOR UPTIME UPTIME",
                            " IEF237I JES2 ALLOCATED TO STDPARM",
                            " IEF237I DMY  ALLOCATED TO STDIN",
                            " IEF237I JES2 ALLOCATED TO STDOUT",
                            " IEF237I JES2 ALLOCATED TO STDERR",
                            " IEF142I UPTIME UPTIME - STEP WAS EXECUTED - COND CODE 0000",
                            " IEF285I   BPANYAR.UPTIME.JOB01151.D0000101.?           SYSIN         ",
                            " IEF285I   BPANYAR.UPTIME.JOB01151.D0000102.?           SYSOUT        ",
                            " IEF285I   BPANYAR.UPTIME.JOB01151.D0000103.?           SYSOUT        ",
                            " IEF373I STEP/UPTIME  /START 2021342.2024",
                            " IEF032I STEP/UPTIME  /STOP  2021342.2024 ",
                            "         CPU:     0 HR  00 MIN  00.00 SEC    SRB:     0 HR  00 MIN  00.00 SEC    ",
                            "         VIRT:   168K  SYS:   248K  EXT:      132K  SYS:    11744K",
                            "         ATB- REAL:                  1044K  SLOTS:                     0K",
                            "              VIRT- ALLOC:      11M SHRD:       0M",
                            " IEF375I  JOB/UPTIME  /START 2021342.2024",
                            " IEF033I  JOB/UPTIME  /STOP  2021342.2024 ",
                            "         CPU:     0 HR  00 MIN  00.00 SEC    SRB:     0 HR  00 MIN  00.00 SEC    ",
                            " 08:24PM  up 31 day(s), 05:51,  1 users,  load average: 0.00, 0.00, 0.00"
                        ],
                        "ddname": "JESMSGLG",
                        "id": "2",
                        "procstep": "",
                        "record_count": "20",
                        "stepname": "JES2"
                    },
                    {
                        "byte_count": "546",
                        "content": [
                            "1                    J E S 2  J O B  L O G  --  S Y S T E M  I P O 1  --  N O D E  T I V L P 0 2         ",
                            "0 ",
                            " 20.24.59 JOB01151 ---- WEDNESDAY, 08 DEC 2021 ----",
                            " 20.24.59 JOB01151  IRR010I  USERID BPANYAR  IS ASSIGNED TO THIS JOB.",
                            " 20.24.59 JOB01151  $HASP375 JES2     ESTIMATED  LINES EXCEEDED",
                            " 20.24.59 JOB01151  ICH70001I BPANYAR  LAST ACCESS AT 20:24:59 ON WEDNESDAY, DECEMBER 8, 2021",
                            " 20.24.59 JOB01151  $HASP375 UPTIME   ESTIMATED  LINES EXCEEDED",
                            " 20.24.59 JOB01151  $HASP373 UPTIME   STARTED - INIT K    - CLASS A        - SYS IPO1",
                            " 20.24.59 JOB01151  -                                                --TIMINGS (MINS.)--            ----PAGING COUNTS---",
                            " 20.24.59 JOB01151  -JOBNAME  STEPNAME PROCSTEP    RC   EXCP   CONN    TCB    SRB  CLOCK   SERV  PG  PAGE  SWAP   VIO SWAPS",
                            " 20.24.59 JOB01151  -UPTIME   UPTIME               00     40      1    .00    .00     .0    122   0     0     0     0     0",
                            " 20.24.59 JOB01151  -UPTIME   ENDED.  NAME-UPTIME - JRM         TOTAL TCB CPU TIME=   .00  TOTAL ELAPSED TIME=    .0",
                            " 20.24.59 JOB01151  $HASP395 UPTIME   ENDED - RC=0000",
                            "0------ JES2 JOB STATISTICS ------",
                            "-  08 DEC 2021 JOB EXECUTION DATE",
                            "-           12 CARDS READ",
                            "-           58 SYSOUT PRINT RECORDS",
                            "-            0 SYSOUT PUNCH RECORDS",
                            "-            7 SYSOUT SPOOL KBYTES",
                            "-         0.00 MINUTES EXECUTION TIME",
                            "         1 //UPTIME    JOB (T043JM,JM00,1,0,0,0),'UPTIME - JRM',                   JOB01151",
                            "           //             MSGCLASS=X,MSGLEVEL=1,NOTIFY=&SYSUID                             ",
                            "           //*                                                                             ",
                            "           //* PRINT USS COMMAND ON JOB OUTPUT                                             ",
                            "           //*                                                                             ",
                            "           IEFC653I SUBSTITUTION JCL - (T043JM,JM00,1,0,0,0),'UPTIME - JRM',MSGCLASS=X,MSGLEVEL=1,NOTIFY=BPANYAR",
                            "         2 //UPTIME  EXEC PGM=BPXBATCH                                                     ",
                            "         3 //STDPARM DD *                                                                  ",
                            "         4 //STDIN  DD DUMMY                                                               ",
                            "         5 //STDOUT DD SYSOUT=*                                                            ",
                            "         6 //STDERR DD SYSOUT=*                                                            ",
                            "         7 //                                                                              ",
                            " ICH70001I BPANYAR  LAST ACCESS AT 20:24:59 ON WEDNESDAY, DECEMBER 8, 2021",
                            " IEFA111I UPTIME IS USING THE FOLLOWING JOB RELATED SETTINGS:",
                            "          SWA=BELOW,TIOT SIZE=32K,DSENQSHR=DISALLOW,GDGBIAS=JOB",
                            " IEF236I ALLOC. FOR UPTIME UPTIME",
                            " IEF237I JES2 ALLOCATED TO STDPARM",
                            " IEF237I DMY  ALLOCATED TO STDIN",
                            " IEF237I JES2 ALLOCATED TO STDOUT",
                            " IEF237I JES2 ALLOCATED TO STDERR",
                            " IEF142I UPTIME UPTIME - STEP WAS EXECUTED - COND CODE 0000",
                            " IEF285I   BPANYAR.UPTIME.JOB01151.D0000101.?           SYSIN         ",
                            " IEF285I   BPANYAR.UPTIME.JOB01151.D0000102.?           SYSOUT        ",
                            " IEF285I   BPANYAR.UPTIME.JOB01151.D0000103.?           SYSOUT        ",
                            " IEF373I STEP/UPTIME  /START 2021342.2024",
                            " IEF032I STEP/UPTIME  /STOP  2021342.2024 ",
                            "         CPU:     0 HR  00 MIN  00.00 SEC    SRB:     0 HR  00 MIN  00.00 SEC    ",
                            "         VIRT:   168K  SYS:   248K  EXT:      132K  SYS:    11744K",
                            "         ATB- REAL:                  1044K  SLOTS:                     0K",
                            "              VIRT- ALLOC:      11M SHRD:       0M",
                            " IEF375I  JOB/UPTIME  /START 2021342.2024",
                            " IEF033I  JOB/UPTIME  /STOP  2021342.2024 ",
                            "         CPU:     0 HR  00 MIN  00.00 SEC    SRB:     0 HR  00 MIN  00.00 SEC    ",
                            " 08:24PM  up 31 day(s), 05:51,  1 users,  load average: 0.00, 0.00, 0.00"
                        ],
                        "ddname": "JESJCL",
                        "id": "3",
                        "procstep": "",
                        "record_count": "12",
                        "stepname": "JES2"
                    },
                    {
                        "byte_count": "1194",
                        "content": [
                            "1                    J E S 2  J O B  L O G  --  S Y S T E M  I P O 1  --  N O D E  T I V L P 0 2         ",
                            "0 ",
                            " 20.24.59 JOB01151 ---- WEDNESDAY, 08 DEC 2021 ----",
                            " 20.24.59 JOB01151  IRR010I  USERID BPANYAR  IS ASSIGNED TO THIS JOB.",
                            " 20.24.59 JOB01151  $HASP375 JES2     ESTIMATED  LINES EXCEEDED",
                            " 20.24.59 JOB01151  ICH70001I BPANYAR  LAST ACCESS AT 20:24:59 ON WEDNESDAY, DECEMBER 8, 2021",
                            " 20.24.59 JOB01151  $HASP375 UPTIME   ESTIMATED  LINES EXCEEDED",
                            " 20.24.59 JOB01151  $HASP373 UPTIME   STARTED - INIT K    - CLASS A        - SYS IPO1",
                            " 20.24.59 JOB01151  -                                                --TIMINGS (MINS.)--            ----PAGING COUNTS---",
                            " 20.24.59 JOB01151  -JOBNAME  STEPNAME PROCSTEP    RC   EXCP   CONN    TCB    SRB  CLOCK   SERV  PG  PAGE  SWAP   VIO SWAPS",
                            " 20.24.59 JOB01151  -UPTIME   UPTIME               00     40      1    .00    .00     .0    122   0     0     0     0     0",
                            " 20.24.59 JOB01151  -UPTIME   ENDED.  NAME-UPTIME - JRM         TOTAL TCB CPU TIME=   .00  TOTAL ELAPSED TIME=    .0",
                            " 20.24.59 JOB01151  $HASP395 UPTIME   ENDED - RC=0000",
                            "0------ JES2 JOB STATISTICS ------",
                            "-  08 DEC 2021 JOB EXECUTION DATE",
                            "-           12 CARDS READ",
                            "-           58 SYSOUT PRINT RECORDS",
                            "-            0 SYSOUT PUNCH RECORDS",
                            "-            7 SYSOUT SPOOL KBYTES",
                            "-         0.00 MINUTES EXECUTION TIME",
                            "         1 //UPTIME    JOB (T043JM,JM00,1,0,0,0),'UPTIME - JRM',                   JOB01151",
                            "           //             MSGCLASS=X,MSGLEVEL=1,NOTIFY=&SYSUID                             ",
                            "           //*                                                                             ",
                            "           //* PRINT USS COMMAND ON JOB OUTPUT                                             ",
                            "           //*                                                                             ",
                            "           IEFC653I SUBSTITUTION JCL - (T043JM,JM00,1,0,0,0),'UPTIME - JRM',MSGCLASS=X,MSGLEVEL=1,NOTIFY=BPANYAR",
                            "         2 //UPTIME  EXEC PGM=BPXBATCH                                                     ",
                            "         3 //STDPARM DD *                                                                  ",
                            "         4 //STDIN  DD DUMMY                                                               ",
                            "         5 //STDOUT DD SYSOUT=*                                                            ",
                            "         6 //STDERR DD SYSOUT=*                                                            ",
                            "         7 //                                                                              ",
                            " ICH70001I BPANYAR  LAST ACCESS AT 20:24:59 ON WEDNESDAY, DECEMBER 8, 2021",
                            " IEFA111I UPTIME IS USING THE FOLLOWING JOB RELATED SETTINGS:",
                            "          SWA=BELOW,TIOT SIZE=32K,DSENQSHR=DISALLOW,GDGBIAS=JOB",
                            " IEF236I ALLOC. FOR UPTIME UPTIME",
                            " IEF237I JES2 ALLOCATED TO STDPARM",
                            " IEF237I DMY  ALLOCATED TO STDIN",
                            " IEF237I JES2 ALLOCATED TO STDOUT",
                            " IEF237I JES2 ALLOCATED TO STDERR",
                            " IEF142I UPTIME UPTIME - STEP WAS EXECUTED - COND CODE 0000",
                            " IEF285I   BPANYAR.UPTIME.JOB01151.D0000101.?           SYSIN         ",
                            " IEF285I   BPANYAR.UPTIME.JOB01151.D0000102.?           SYSOUT        ",
                            " IEF285I   BPANYAR.UPTIME.JOB01151.D0000103.?           SYSOUT        ",
                            " IEF373I STEP/UPTIME  /START 2021342.2024",
                            " IEF032I STEP/UPTIME  /STOP  2021342.2024 ",
                            "         CPU:     0 HR  00 MIN  00.00 SEC    SRB:     0 HR  00 MIN  00.00 SEC    ",
                            "         VIRT:   168K  SYS:   248K  EXT:      132K  SYS:    11744K",
                            "         ATB- REAL:                  1044K  SLOTS:                     0K",
                            "              VIRT- ALLOC:      11M SHRD:       0M",
                            " IEF375I  JOB/UPTIME  /START 2021342.2024",
                            " IEF033I  JOB/UPTIME  /STOP  2021342.2024 ",
                            "         CPU:     0 HR  00 MIN  00.00 SEC    SRB:     0 HR  00 MIN  00.00 SEC    ",
                            " 08:24PM  up 31 day(s), 05:51,  1 users,  load average: 0.00, 0.00, 0.00"
                        ],
                        "ddname": "JESYSMSG",
                        "id": "4",
                        "procstep": "",
                        "record_count": "21",
                        "stepname": "JES2"
                    },
                    {
                        "byte_count": "108",
                        "content": [
                            "1                    J E S 2  J O B  L O G  --  S Y S T E M  I P O 1  --  N O D E  T I V L P 0 2         ",
                            "0 ",
                            " 20.24.59 JOB01151 ---- WEDNESDAY, 08 DEC 2021 ----",
                            " 20.24.59 JOB01151  IRR010I  USERID BPANYAR  IS ASSIGNED TO THIS JOB.",
                            " 20.24.59 JOB01151  $HASP375 JES2     ESTIMATED  LINES EXCEEDED",
                            " 20.24.59 JOB01151  ICH70001I BPANYAR  LAST ACCESS AT 20:24:59 ON WEDNESDAY, DECEMBER 8, 2021",
                            " 20.24.59 JOB01151  $HASP375 UPTIME   ESTIMATED  LINES EXCEEDED",
                            " 20.24.59 JOB01151  $HASP373 UPTIME   STARTED - INIT K    - CLASS A        - SYS IPO1",
                            " 20.24.59 JOB01151  -                                                --TIMINGS (MINS.)--            ----PAGING COUNTS---",
                            " 20.24.59 JOB01151  -JOBNAME  STEPNAME PROCSTEP    RC   EXCP   CONN    TCB    SRB  CLOCK   SERV  PG  PAGE  SWAP   VIO SWAPS",
                            " 20.24.59 JOB01151  -UPTIME   UPTIME               00     40      1    .00    .00     .0    122   0     0     0     0     0",
                            " 20.24.59 JOB01151  -UPTIME   ENDED.  NAME-UPTIME - JRM         TOTAL TCB CPU TIME=   .00  TOTAL ELAPSED TIME=    .0",
                            " 20.24.59 JOB01151  $HASP395 UPTIME   ENDED - RC=0000",
                            "0------ JES2 JOB STATISTICS ------",
                            "-  08 DEC 2021 JOB EXECUTION DATE",
                            "-           12 CARDS READ",
                            "-           58 SYSOUT PRINT RECORDS",
                            "-            0 SYSOUT PUNCH RECORDS",
                            "-            7 SYSOUT SPOOL KBYTES",
                            "-         0.00 MINUTES EXECUTION TIME",
                            "         1 //UPTIME    JOB (T043JM,JM00,1,0,0,0),'UPTIME - JRM',                   JOB01151",
                            "           //             MSGCLASS=X,MSGLEVEL=1,NOTIFY=&SYSUID                             ",
                            "           //*                                                                             ",
                            "           //* PRINT USS COMMAND ON JOB OUTPUT                                             ",
                            "           //*                                                                             ",
                            "           IEFC653I SUBSTITUTION JCL - (T043JM,JM00,1,0,0,0),'UPTIME - JRM',MSGCLASS=X,MSGLEVEL=1,NOTIFY=BPANYAR",
                            "         2 //UPTIME  EXEC PGM=BPXBATCH                                                     ",
                            "         3 //STDPARM DD *                                                                  ",
                            "         4 //STDIN  DD DUMMY                                                               ",
                            "         5 //STDOUT DD SYSOUT=*                                                            ",
                            "         6 //STDERR DD SYSOUT=*                                                            ",
                            "         7 //                                                                              ",
                            " ICH70001I BPANYAR  LAST ACCESS AT 20:24:59 ON WEDNESDAY, DECEMBER 8, 2021",
                            " IEFA111I UPTIME IS USING THE FOLLOWING JOB RELATED SETTINGS:",
                            "          SWA=BELOW,TIOT SIZE=32K,DSENQSHR=DISALLOW,GDGBIAS=JOB",
                            " IEF236I ALLOC. FOR UPTIME UPTIME",
                            " IEF237I JES2 ALLOCATED TO STDPARM",
                            " IEF237I DMY  ALLOCATED TO STDIN",
                            " IEF237I JES2 ALLOCATED TO STDOUT",
                            " IEF237I JES2 ALLOCATED TO STDERR",
                            " IEF142I UPTIME UPTIME - STEP WAS EXECUTED - COND CODE 0000",
                            " IEF285I   BPANYAR.UPTIME.JOB01151.D0000101.?           SYSIN         ",
                            " IEF285I   BPANYAR.UPTIME.JOB01151.D0000102.?           SYSOUT        ",
                            " IEF285I   BPANYAR.UPTIME.JOB01151.D0000103.?           SYSOUT        ",
                            " IEF373I STEP/UPTIME  /START 2021342.2024",
                            " IEF032I STEP/UPTIME  /STOP  2021342.2024 ",
                            "         CPU:     0 HR  00 MIN  00.00 SEC    SRB:     0 HR  00 MIN  00.00 SEC    ",
                            "         VIRT:   168K  SYS:   248K  EXT:      132K  SYS:    11744K",
                            "         ATB- REAL:                  1044K  SLOTS:                     0K",
                            "              VIRT- ALLOC:      11M SHRD:       0M",
                            " IEF375I  JOB/UPTIME  /START 2021342.2024",
                            " IEF033I  JOB/UPTIME  /STOP  2021342.2024 ",
                            "         CPU:     0 HR  00 MIN  00.00 SEC    SRB:     0 HR  00 MIN  00.00 SEC    ",
                            " 08:24PM  up 31 day(s), 05:51,  1 users,  load average: 0.00, 0.00, 0.00"
                        ],
                        "ddname": "STDOUT",
                        "id": "102",
                        "procstep": "",
                        "record_count": "1",
                        "stepname": "UPTIME"
                    }
                ],
                "job_id": "JOB01151",
                "job_name": "UPTIME",
                "owner": "BPANYAR",
                "ret_code": {
                    "code": 0,
                    "msg": "CC 0000",
                    "msg_code": "0000",
                    "msg_txt": "",
                    "steps": [
                        {
                            "step_cc": "0000",
                            "step_name": "UPTIME"
                        },
                        {
                            "step_cc": "0000",
                            "step_name": "UPTIME"
                        },
                        {
                            "step_cc": "0000",
                            "step_name": "UPTIME"
                        },
                        {
                            "step_cc": "0000",
                            "step_name": "UPTIME"
                        }
                    ]
                },
                "subsystem": "IPO1",
                "system": ""
            }
        ],
        "message": {
            "stdout": "Submit JCL operation succeeded with id of JOB01151."
```

**Comment by ddimatos** [comment]
Tracked in JIRA 8885

**Comment by rexemin** [comment]
This bug was addressed by #318. Validated that the buggy behavior stops ocurring on the current state of the dev branch.

**Comment by ddimatos** [comment]
Thank you @rexemin for evaluating this. 
@itsBryantP given this is fixed in the development branch with no plan to backport as its not a simple migration to backport its a nearly rewrite and that this is duplicate content it feels acceptable to point users to our 1.5.0-x versions for this fix.

We will document this as a known issue in 1.4.0-x.
